FWCore/Framework/interface/InputSource.h

```diff
@@ -184,7 +189,7 @@ namespace edm {
     }
     
     /// Returns nullptr if no resource shared between the Source and a DelayedReader
-    SharedResourcesAcquirer* resourceSharedWithDelayedReader() const;
```

Comment:
OK to return pointer to non-const in const function because SharedResourcesAcquire is mutable